specs/core/0_beacon-chain.md

```diff
@@ -1262,61 +1265,63 @@ For each `attestation` in `block.body.attestations`:
 * If `block.candidate_pow_receipt_root` is `x.candidate_pow_receipt_root` for some `x` in `state.candidate_pow_receipt_roots`, set `x.votes += 1`.
 * Otherwise, append to `state.candidate_pow_receipt_roots` a new `CandidatePoWReceiptRootRecord(candidate_pow_receipt_root=block.candidate_pow_receipt_root, votes=1)`.
 
-### Special objects
-
-* Verify that the quantity of each type of object in `block.body.specials` is less than or equal to its maximum (see table at the top).
-* Verify that objects are sorted in order of `kind`. That is, `block.body.specials[i+1].kind >= block.body.specials[i].kind` for `0 <= i < len(block.body.specials-1)`.
+### Operations
```

Comment:
"Operations" feels a bit awkward. What about "beacon transaction"?

Reply:
I'm not totally against `Beacon transaction`, but @vbuterin wanted to reserve that term exclusively for user level activity.

I think operations is better here than "special" or "objects".

The following seems natural to me
- Verify the block header
- Verify and perform block operations
- Transit epoch if necessary